src/test/java/io/fusionauth/samlv2/service/DefaultSAMLv2ServiceTest.java

```diff
@@ -276,6 +277,33 @@ public void buildLogoutResponse(Binding binding) throws Exception {
     }
   }
 
+  @Test
+  public void buildPostAuthnRequest_signatureFollowsIssuer() throws Exception {
```

Comment:
Let's test with each possible type:
- `AuthenticationRequest`
- `LogoutRequest`
- `LogoutResponse`

Reply:
I didn't notice that these other types called the same method. Thanks!